### PR TITLE
feat: added notification call to boot endpoint

### PR DESCRIPTION
**Pull request by rebelchris**
Added notification endpoint to boot call including our usual redis caching of the notification count.

Tested on extension:
Screenshot 2022-11-30 at 14 43 10

WT-784

### CODE REVIEW

**Review by idoshamun — APPROVED** [review]
Looks good but please check my comments before merging (also I guess we need to merge API first)

File: src/routes/boot.js [hunk 1]
```diff
@@ -214,6 +214,19 @@ const getSettings = async (ctx) => {
   return settings;
 };
 
+const getNotifications = async (ctx) => {
```
Comment (el) [inline src/routes/boot.js:217]: I understand we use js in the backend, should we use JSDoc to add a return type?
```
/**
 * @typedef {Object} Notifications
 * @property {number} notificationCount - Number of unread notifications
 */
 
/**
 * Returns user notifications object
 * @returns {Notifications}
 */
 const getNotifications = async (ctx) => {
```

**Review by el — APPROVED** [review]

**Review by sshanzel — APPROVED** [review]
Spotless! ✨

**Review by idoshamun — APPROVED** [review]

File: src/routes/boot.js (hunk 1 above)
Comment (rebelchris) [reply src/routes/boot.js:217]: Thanks for the feedback, for now out of scope but we can add this to the backlog to look into for the future.
(I'll add it)

**Pull request merged**